images/base/Dockerfile

```diff
@@ -43,20 +43,23 @@ USER ${NB_UID}
 # Set environment variable for Conda cache directory inside the container
 ENV CONDA_PKGS_DIRS=/opt/conda/pkgs
 
-RUN conda install -n base conda-libmamba-solver && \
-  conda config --set solver libmamba
+# Uninstall default R version
```

Comment:
When trying to update to installed R version it fails because some packages (e.g: rpy2, r-askpass) could not be updated to match R 4.4.*, that's why we remove instead of update the previous version.